proxy-socket-core/src/main/java/net/airvantage/proxysocket/core/ProxyProtocolException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * MIT License
+ * Copyright (c) 2025 Semtech
+ */
+package net.airvantage.proxysocket.core;
+
+public class ProxyProtocolException extends Exception {
```

Comment:
The serializable class ProxyProtocolException does not declare a static final serialVersionUID field of type long

We should add something like : 
 ```java
 private static final long serialVersionUID = 1L;
 ```